rectools/model_selection/cross_validate.py

```diff
@@ -24,6 +25,26 @@
 from .splitter import Splitter
 
 
+@contextmanager
+def compute_timing(label: str, timings: tp.Optional[tp.Dict[str, float]] = None) -> tp.Iterator[None]:
+    """
+    Context manager to compute timing for a code block.
+
+    Parameters
+    ----------
+    label : str
+        Label to store the timing result in the timings dictionary.
+    timings : dict, optional
+        Dictionary to store the timing results. If None, timing is not recorded.
+    """
+    if timings is not None:
+        start_time = time.time()
```

Comment:
Please use `time.perf_counter` instead, it's more correct for measuring time intervals